### PR TITLE
Updating to WebDriver 3.14

### PR DESCRIPTION
**Pull request by barancev**

### CODE REVIEW

**Comment by kevingy** [comment]
@Teun I'll let you handle this pull request. You are more familiar with the WebDriver than I am.

**Comment by Teun** [comment]
I will have a look and leave some comments were appropriate, but frankly, I hardly feel ownership of this library anymore. I started it many years ago and have not used it myself for at least three years. I don't even have a development environment available were I can compile and test.

So maybe someone else feels like taking some responsibility here? Maybe Alexei, who apparently want the library to at least be updated to later versions of dependencies, can step in?

File: SimpleBrowser.WebDriver/SimpleBrowser.WebDriver.csproj [hunk 1]
```diff
@@ -33,13 +33,14 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Moq, Version=4.2.1510.2205, Culture=neutral, PublicKeyToken=69f491c39445e920, processorArchitecture=MSIL">
```
Comment (Teun) [inline SimpleBrowser.WebDriver/SimpleBrowser.WebDriver.csproj]: This seems strange: why would the library need a dependency on Moq. That should only be used in the Test project.

Comment (barancev) [reply SimpleBrowser.WebDriver/SimpleBrowser.WebDriver.csproj]: No idea, it was there before, I've just updated the version.

**Comment by barancev** [comment]
I have no plan to take ownership of this project, sorry, my primary area of interests is not related to .Net, this PR is just a side-effect of a short contract with a customer that wanted .Net, but I'm already out.

File: SimpleBrowser.WebDriver/SimpleBrowser.WebDriver.csproj (hunk 1 above)
Comment (Teun) [reply SimpleBrowser.WebDriver/SimpleBrowser.WebDriver.csproj]: No, it wasn't there. Must have happened as part of the update step. No problem. I'll take it out and try to run a CI build on the branch.

**Comment by Teun** [comment]
Right. Fair enough, one can always hope.

**Comment by Teun** [comment]
So I merged this pull request into a separate branch (barancev-master) and ran the CI build over it (this was last done very long ago, so I had to fix some things in the environment as well).

It has 4 failing unit tests. You can see the full build result here: https://teamcity.jetbrains.com/viewLog.html?buildId=1643817&buildTypeId=bt1121&tab=buildResultsDiv

Not sure what to do with this. As the code does hardly involve any real changes, I assume that these failing tests are caused by changes in the underlying dependencies (Selenium? Moq?). We can just mute these tests for now and publish a new version. That will at least allow people to work with it using up-to-date SimpleBrowser and Selenium.

**Comment by Teun** [comment]
Pff. That took more than I thought to get all those CI steps up-to-date and running again. But it is fine now. New version has been published as 0.5.43.

Why the 4 unit tests are failing, I don't know. I would nee a .NET 4 environment to figure it out.

**Pull request merged**